### PR TITLE
Update constants.js

### PR DESCRIPTION
**Pull request by WallyThePenguin**
IMPORTANT: Please do not create a Pull Request without creating an issue first.

Put closes #XXXX in your comment to auto-close the issue that your PR fixes (if such).

Please provide enough information so that others can review your pull request:

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos etc...

### CODE REVIEW

File: commands/Fun/giraffefacts.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+*/
+
+const { Command } = require('klasa');
+const { giraffefacts } = require('../../array.js')
```
Comment (Skillz4Killz) [inline commands/Fun/giraffefacts.js]: array.js??

File: commands/Fun/giraffefacts.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, { description: 'Gives you a random giraffe fact.' });
+	}
+
+	run(message) {
+		return message.sendMessage(facts[Math.floor(Math.random() * giraffefacts.length)]);
```
Comment (Skillz4Killz) [inline commands/Fun/giraffefacts.js]: giraffefacts[Math.floor......

File: commands/Fun/snakefacts.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+*/
+
+const { Command } = require('klasa');
+const { snakefacts } = require('../../array.js')
```
Comment (Skillz4Killz) [inline commands/Fun/snakefacts.js]: array.js???

File: commands/Fun/snakefacts.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, { description: 'Gives you a random snake fact.' });
+	}
+
+	run(message) {
+		return message.sendMessage(facts[Math.floor(Math.random() * snakefacts.length)]);
```
Comment (Skillz4Killz) [inline commands/Fun/snakefacts.js]: snakefacts[Math....

**Comment by WallyThePenguin** [comment]
push

**Pull request merged**